src/datatypes/colorizer.md

```diff
@@ -31,7 +33,8 @@ A linear gradient linearly interpolates values within breakpoints of a color tab
     }
   ],
   "noDataColor": [0, 0, 0, 0],
-  "defaultColor": [0, 0, 0, 0]
+  "overColor": [255, 0, 0, 255],
+  "underColor": [0, 0, 255, 255]
```

Comment:
Are these reasonable over and under colors?

You have a gradient from white to black and then suddenly it is blue? You could continue with white or black, for instance.